specification/monitor/resource-manager/readme.go.md

```diff
@@ -48,6 +48,7 @@ batch:
   - tag: package-2021-05-metrics
   - tag: package-2021-08-scheduledqueryrules
   - tag: package-2022-02-01-preview-only
+  - tag: package-2023-02-01-only
```

Comment:
We no longer support Go track 1 SDK, therefore no changes should be made for this file. Could we please revert this change?